.github/workflows/android.yml

```diff
@@ -0,0 +1,70 @@
+name: android
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
+    if: github.event.pull_request.draft == false
+
+    strategy:
+      matrix:
+        arch: [armeabi-v7a, arm64-v8a, x86, x86_64]
```

Comment:
for my own culture, which of these arch do we actually care about ?

Reply:
most of devices are `arm64-v8a` nowadays, but cheap ones are `armeabi-v7a`.  
x86 are less popular but they exist and it's basically free for us to support.